lib/Sagiri.js

```diff
@@ -31,7 +31,7 @@ class SauceHandler {
 
     /**
      * Gets the source and outputs it in your preferred output type
-     * 
+     *
```

Comment:
why the extra spaces?


clam down with whitespace choc

Reply:
its how atom imported it?